config/secrets.yml

```diff
@@ -17,20 +17,65 @@ development:
   ezid_password: <%= ENV['EZID_PASSWORD'] %>
```

Comment:
You could create a yaml var for default here and import it under all the environment namespaces too if you want to reduce some code here. Considering staging/prod are exactly the same (or dev/test are very similar)